server/db/models/contact_lists.js

```diff
@@ -0,0 +1,22 @@
+import knexModule from 'knex';
+import bookshelfModule from 'bookshelf';
+import Contact from './contacts';
+import Campaign from './campaigns';
+import { development as devconfig } from '../../../knexfile';
+
+const knex = knexModule(devconfig);
+const bookshelf = bookshelfModule(knex);
+
+
+export default function ContactList(bookshelfObj) {
+  return bookshelfObj.Model.extend({
+    tableName: 'contact_lists',
+    hasTimestamps: true,
+    contacts() {
+      return this.belongsToMany(Contact(bookshelf));
```

Comment:
I believe `hasMany` refers to one-to-many relationships whereas `belongsToMany` is used for many-to-many relationships - http://bookshelfjs.org/#Model-instance-belongsToMany